### PR TITLE
(yagan cluster) add lhn subnet

### PR DESCRIPTION
**Pull request by jhoblitt**
Closes #740

### CODE REVIEW

**Review by cbarria — COMMENTED** [review]
its more clean and the test are improved from the test on ruka, Looks great, just the additional interface question.

File: hieradata/cluster/yagan.yaml [hunk 1]
```diff
@@ -29,10 +30,34 @@ network::interfaces_hash:
...
+    nozeroconf: "yes"
+    onboot: "yes"
+    type: "none"
+    vlan: "yes"
+    bridge: "br1800"
+  br1800:
+    bootproto: "none"
+    onboot: "yes"
+    type: "Bridge"
+  lhnrouting:  # dummy interface to kludge around EL7 NM
```
[the remark this answers is not in the document]
Comment (jhoblitt) [reply hieradata/cluster/yagan.yaml:45]: I'm marking this as resolved as it was discussed at this morning's meeting.

**Review by cbarria — APPROVED** [review]
Looks Great 👍🏻

**Comment by jhoblitt** [comment]
Rebased on master after the merge of #748

**Pull request merged**